wemake_python_styleguide/visitors/ast/complexity/counts.py

```diff
@@ -212,12 +212,12 @@ def _count_conditions(self, node: ast.BoolOp) -> int:
 
     def _check_conditions(self, node: ast.BoolOp) -> None:
         conditions_count = self._count_conditions(node)
-        if conditions_count > self._max_conditions:
+        if conditions_count > self.MAX_CONDITIONS:
```

Comment:
Not `self.MAX_CONDITIONS`, but just `MAX_CONDITIONS`
Because we import this name from `constants` at the top level.